frontend/src/helpers/campCollaborationDisplayName.js

```diff
@@ -0,0 +1,18 @@
+import i18n from '@/plugins/i18n'
+
+/**
+ * Returns a display name for a camp collaboration based on its status
+ */
+export default function (campCollaboration) {
+  if (campCollaboration.user === null) {
+    return campCollaboration.inviteEmail
```

Comment:
könnte in diesem fall auch inactive sein, aber kann man auch später lösen.

Reply:
Done in https://github.com/ecamp/ecamp3/pull/2404/commits/cc58375314d3b06d74947d53bef04730f97c194a